wafer/schedule/admin.py

```diff
@@ -187,6 +189,13 @@ def changelist_view(self, request, extra_context=None):
 
 
 class SlotAdminForm(forms.ModelForm):
+
+    additional = forms.IntegerField(min_value=0, max_value=30, required=False,
```

Comment:
Good point - it will currently show up on the edit option.

It should be possible to hide it by tweaking SlotAdmin.get_form - I'll need to think a bit about how best to implement that.